content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/unity.md

```diff
@@ -28,7 +28,7 @@ further_reading:
 ## Overview
 
 {{< beta-callout url="#" btn_hidden="true" >}}
-Unity Monitoring is in private beta. To request access, reach out to Datadog support.
+Unity Monitoring is in public beta. You can get started right away and [create a Unity application][1] on Datadog.
```

Comment:
```suggestion
Unity Monitoring is in public beta.
```

Reply:
I get the sentiment, but to follow our current docs style, we keep the beta callouts simple and straightforward. :)